website/docs/r/policy_set.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_sentinel_policy"
+sidebar_current: "docs-resource-tfe-sentinel-policy"
+description: |-
+  Sentinel Policy as Code is an embedded policy as code framework integrated with Terraform Enterprise.
+---
+
+# tfe_policy_set
+
+Sentinel Policy as Code is an embedded policy as code framework integrated
+with Terraform Enterprise.
+
+Policy sets are configured on a per-organization level, and allow organization
+owners to choose which policies are enforced on which workspaces.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_policy_set" "test" {
+  name = "my-policy-set"
+  description = "A brand new policy set"
+  global = false
+  organization = "my-org-name"
+
+  policies = ["${tfe_sentinel_policy.test.id}"]
+  workspaces = ["${tfe_workspace.test.id}"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Name of the policy set.
+* `description` - (Optional) A description of the policy set's purpose.
+* `global` - (Optional) Whether or not policies in this set will apply to
+  all workspaces. Defaults to `false`.
+* `organization` - (Required) Name of the organization.
+* `policies` - (Required) A list of Sentinel policy IDs.
+* `workspaces` - (Required) A list of workspace IDs. If the policy set is
+  `global`, however, this value _must not_ be provided.
+
+## Attributes Reference
+
+* `id` - The ID of the policy set.
+
+## Import
+
+Policy sets can be imported by concatenating the `organization name` and
+`resource id`, e.g.
+
+```shell
+terraform import tfe_policy_set.test my-org-name/polset-wAs3zYmWAhYK7peR
+```
```

Comment:
There is a PR (#32) to update the import documentation just a tiny bit, so good to update this so that its inline with the upcoming changes.

Reply:
PR #32 is merged, so please use the same format here (it was changed again just before it god merged).